src/data/transform.ts

```diff
@@ -367,7 +411,7 @@ const bindLinearElementToElement = (
 class ElementStore {
   excalidrawElements = new Map<string, ExcalidrawElement>();
 
-  add = (ele?: ExcalidrawElement) => {
+  add = (ele?: ExcalidrawElement, originalId?: string) => {
```

Comment:
unused `originalId`?

Reply:
Eagle's eye, I didn't get any warning for this unused variable :/